Jenkinsfile

```diff
@@ -265,10 +265,16 @@ pipeline {
             }
 
             steps {
-                sh '''
```

Comment:
commit-msg: this did not work for private repos that require authentication as credentails were not passed in this step.

I believe there is some word missing in this bit of the sentence.

Reply:
Hmm, not really. What do you miss?